Segmentio/Source/Cells/SegmentioCell.swift

```diff
@@ -190,6 +190,9 @@ class SegmentioCell: UICollectionViewCell {
                 multiplier: 1,
                 constant: 0
         )
+        if #available(iOS 9, *) {
+            segmentTitleLabel.leadingAnchor.constraint(equalTo: containerView.leadingAnchor, constant: 5).isActive = true
```

Comment:
Sure. It happens on prior iOS 9 devices too.

![screen shot 2016-12-31 at 13 10 48](https://cloud.githubusercontent.com/assets/1775157/21577443/ddb3320c-cf5a-11e6-9618-582f8fffbc96.png)
![screen shot 2016-12-31 at 13 12 07](https://cloud.githubusercontent.com/assets/1775157/21577444/ddb3a4a8-cf5a-11e6-8ebe-90f5c0122d93.png)